Makefile

```diff
@@ -30,7 +31,8 @@ format-shell:
 
 .PHONY: format-solidity
 format-solidity:
-	"$$(npm config get prefix)/bin/prettier" --print-width 100 --write \
+	"$(NPM_DIR)/bin/prettier" --print-width 100 --write \
+		--plugin="$(NPM_DIR)/lib/node_modules/prettier-plugin-solidity/src/index.js" \
```

Comment:
With the 3.0.0 release of `prettier`, plugins are no longer loaded by default. You have to use this not-so-pretty flag or use a configuration file.